lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -752,6 +755,23 @@ function dosomething_campaign_add_signup_data_form_vars(&$node) {
   }
 }
 
+/*
+ * Adds flag to the node that tells us if we need to show the organ donation modal after sign up.
+ */
+function dosomething_campaign_add_oragan_donation_vars(&$node) {
+  $organ_registration_turned_on = dosomething_helpers_get_variable('node', $node->nid, 'register_organ_donors');
+
+  $sid = dosomething_signup_exists($node->nid);
```

Comment:
I am a jerk, but let's please make sure we're spelling `organ` and `donor` correctly everywhere: http://cl.ly/3B1c0t3r1g2K